phpstan.neon

```diff
@@ -83,6 +83,12 @@ parameters:
         - message: '#Call to an undefined method ApiPlatform\\Core\\DataPersister\\ContextAwareDataPersisterInterface\:\:(getSimpleDataPersisters|getDataPersisters)\(\)#'
           path: packages/EasyCore/src/Bridge/Symfony/ApiPlatform/DataPersister/TraceableChainSimpleDataPersister.php
 
+        - message: '#Parameter \#1 \$entityManager of method class@anonymous/EasyCore/tests/Bridge/Symfony/Traits/EntityManagerAwareTraitTest\.php\:15\:\:setEntityManager\(\) expects Doctrine\\ORM\\EntityManagerInterface, Mockery\\MockInterface given#'
```

Comment:
Do we really need these? You've already added PHPDocs for the corresponding variables. Should be enough.

Reply:
`vendor/bin/phpstan analyse packages/EasyCore/tests/Bridge/Symfony/Traits --error-format symplify
Note: Using configuration file /mnt/d/eonx/easy-monorepo/phpstan.neon.
 2/2 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  packages/EasyCore/tests/Bridge/Symfony/Traits/EntityManagerAwareTraitTest.php:21
 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  - '#Parameter \#1 \$entityManager of method class@anonymous/EntityManagerAwareTraitTest\.php\:15\:\:setEntityManager\(\) expects Doctrine\\ORM\\EntityManagerInterface, Mockery\\MockInterface given#'
 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  packages/EasyCore/tests/Bridge/Symfony/Traits/TranslatorAwareTraitTest.php:21
 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  - '#Parameter \#1 \$translator of method class@anonymous/TranslatorAwareTraitTest\.php\:15\:\:setTranslator\(\) expects Symfony\\Contracts\\Translation\\TranslatorInterface, Mockery\\MockInterface given#'
 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------`